src/App.js

```diff
@@ -16,7 +17,13 @@ import { dashboardUpdate, isVersionAllowedUpdate } from "./actions/index";
 import { connect } from "react-redux";
 import versionApi from './api/version';
 import helpers from './utils/helpers';
+import MemoryStore from './store';
```

Comment:
Why don't we move the `MemoryStore` to the hathor-wallet-lib?

Reply:
In this case is another problem. The explorer does not need any information saved in the `localStorage`. We are only using some util methods. I think in a lib refactor we should make the storage optional.

Reply:
I agree. But, since you've added a MemoryStore here, isn't it better to just add it to the hathor-wallet-lib and use it from here? We have other repositories using MemoryStore as well.

Reply:
Makes sense. It's not for use in this explorer specifically, it's just in case another project wants a fast MemoryStore without need of implementing it.

Added it to the lib and I am just using here now.